project_template/forms.py

```diff
@@ -173,6 +173,20 @@ class TranscribeOwnedIncomeFromInvestmentsTable(forms.Form):
     count = forms.IntegerField(label="Câte rânduri completate există în tabelul {}?".format(constants.DECLARATION_TABLES['income_investments']))
 
 
+class TranscribeOwnedIncomeFromInvestmentsRowEntry(forms.Form):
+    holder_relationship = forms.ChoiceField(label="Cine este beneficiarul venitului din investitii?", choices=HolderRelationship.return_as_iterable())
+    surname = forms.CharField(label="Care e numele persoanei?")
+    name = forms.CharField(label="Care e prenumele persoanei?")
+    source = forms.CharField(label="Care este sursa venitului?")
+    county = forms.ChoiceField(label="Care este judetul de unde provine sursa de venit?", choices=Counties.return_counties())
+    city = forms.CharField(label="Care este localitatea de unde provine sursa de venit?")
+    commune = forms.CharField(label="Care este comuna de unde provine sursa de venit?")
+    offered_service = forms.CharField(label="Care e serviciul prestat?")
+    position = forms.ChoiceField(label="Care a fost functia detinuta?", choices=Position.return_as_iterable())
```

Comment:
This is redundant. The `offered_service` variable already holds the information. Please remove it.

The header of the table, for this column, is `Serviciul prestat / Obiectul generator de venit`. Incidentally, the "service" is a position that this person holds within an institution. But this is, more abstractly, a "service" (so already covered by the previous variable).